### ISSUE TITLE
gremzeppelin usage?

### ISSUE THREAD

**Issue by MohsenIT** [shikeio/gremzeppelin#1]
Hi,
How can I add your Gremlin interpreter to Zeppelin?

**Comment by boliza** [comment]
Sorry not finish yet

**Comment by MohsenIT** [comment]
Do you have any plan to finish it?

**Comment by MohsenIT** [comment]
Hi,
I cut out with Gremlin and implement my own graph library from zero.
Thanks,
Mohsen

**Comment by wcbdata** [comment]
@MohsenIT - Thanks! Do you have a new project started for the scratch-built graph library? (And do you have plans to implement a Zeppelin interpreter based on it?)

**Comment by MohsenIT** [comment]
No, It is a single purpose graph with an specific instructions. It is not
significant to use it on zeppelin or ... .
